begin.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
***Code Quality: Critical***
This seems to be designed without any use of css or care for responsiveness. This should be refactored and appropriate css should be used.